File: Palindrome.c
#include <stdio.h>
#include <string.h>

int main()
{
    char strl[1000],revstrl[1000];
    gets(strl);
    revstrl = strrev(strl);
    if(strl == revstrl)
    printf("Palindrome ");
    else
    printf("Not Palindrome");
    
}

File: Palindrome.py
def palindrome(strl):
    leng = len(strl)        #input:s = "asdsa"
    revstrl = []            #      palindrome(s)
    for i in range(0, leng):
        revstrl = reversed(strl)
    if list(strl) == list(revstrl):
        print("Palindrome")
    else:
        print("Not Palinfrome")
    
        
